landscape.yml

```diff
@@ -2648,7 +2648,7 @@ landscape:
           - item:
             name: Presto
             homepage_url: https://prestodb.io/
-            repo_url: https://github.com/trinodb/trino
+            repo_url: https://github.com/prestodb/presto
```

Comment:
Shouldn't this be trinodb/trino? Not sure on the relationship between the two projects.

Reply:
No ... Presto and Trino are completely separate projects .. with shared founders and history ..

Reply:
@castrojo, for context: 
* https://trino.io/blog/2020/12/27/announcing-trino.html
* https://trino.io/blog/2022/08/02/leaving-facebook-meta-best-for-trino.html